### DISCUSSION TITLE
Pagination not working with People search

### DISCUSSION THREAD

**Discussion by Ami99049** [microsoft-search/pnp-modern-search#3233] [Q&A]
I am using Microsoft search as a Data source and People as a Entity. I have enabled the versioning but still its not working. I can only see the first page data

**Comment by kasperbolarsen** [comment]
please follow the issue guidelines otherwise we might have to reject your issue

**Reply by Ami99049** [reply]
@kasperbolarsen 
Sorry for the inconvenience 
Please help .. Actually i recently started using this platform and don't have much idea about the format.. please help me..
Issue- pagination not working
Steps
 Add PNP search result webpart
Select Microsoft search as a data source
Select people as entity type
On the pagination settings
Set number of items per page as 5

Current Behaviour
We can see only five results no matter how many results are there matching the search criteria

Expected behaviour
It should show a pagination option if the number of results is more than the current page limit

**Comment by Ami99049** [comment]
@kasperbolarsen 
Sure I will,
But for now can you please look into it

**Comment by kasperbolarsen** [comment]
Sorry, but no. If you don't respect my time and effort I don't see why I should help you. We do this our sparetime and don't get paid by MS or anybody else.

**Comment by Ami99049** [comment]
@kasperbolarsen 
Sorry for the inconvenience 
Please help .. Actually i recently started using this platform and don't have much idea about the format.. please help me..
Issue- pagination not working
Steps
 Add PNP search result webpart
Select Microsoft search as a data source
Select people as entity type
On the pagination settings
Set number of items per page as 5

Current Behaviour
We can see only five results no matter how many results are there matching the search criteria

Expected behaviour
It should show a pagination option if the number of results is more than the current page limit
